evasquez26/angular4-startup
# Added a search bar at the top of the navbar

## Story / Task

[ONE-89](https://bixlabs.atlassian.net/browse/ONE-89)

## What this PR does?

Adds the search bar layout


### Before 

We did not have search bar

### After

- We have a search bar at the top of the navbar
- The navbar is now static scrolling down

### How to test / reproduce

- Run the app
- Observe that now there's a search bar at the top

### TODOs

N/A

### Related PRs

N/A

## Migrations

NO

Thank you!
